src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -59,6 +59,28 @@
 
     private static final Logger logger = LogManager.getLogger(IntervalUtils.class);
 
+    /**
+     * Compare two locations using a {@link htsjdk.samtools.SAMSequenceDictionary} sequence ordering
+     */
+    public static final int compareLocatables(Locatable first, Locatable second, SAMSequenceDictionary dictionary) {
+        int result = 0;
+        if(first != second) {
+            // get the contigs
+            final int firstRefIndex = dictionary.getSequenceIndex(first.getContig());
+            final int secondRefIndex = dictionary.getSequenceIndex(second.getContig());
```

Comment:
If either `firstRefIndex` and `secondRefIndex` is -1 at this point (because `dictionary.getSequenceIndex()` could not find the contigs), throw an `IllegalArgumentException`, since we can't do the comparison in that case.